Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-class PostRideSummaryViewController: UIViewController {
+class PostRideSummaryViewController: PostRideViewController {
     
     private let iconTextSpacing = 10
     private let iconSize = 19
```

Comment:
I would create a `Constants` struct so that you know these are constants:
```
private struct Constants {
    static let iconTextSpacing: CGFloat = 10
    static let iconSize: CGFloat = 19
}
```
Then use `Constants.iconTextSpacing` etc. I see this used everywhere in Volume and it really helps. I would also suggest using this everytime you see yourself repeating some design value such as fonts, etc.